apollo/bags.py

```diff
@@ -86,9 +88,60 @@ def preprocess_source(args):
     pipeline_graph(args, log, ignition)
 
 
+@pause
 def source2bags(args):
+    log = logging.getLogger("sourced2bags")
+    extractors = create_extractors_from_args(args)
+    session_name = "sourced2bags-%s" % uuid4()
     cassandra_utils.configure(args)
-    return repos2bow_entry_template(
-        args,
-        select=lambda: DzhigurdaFiles(args.dzhigurda),
-        cache_hook=lambda: MetadataSaver(args.keyspace, args.tables["meta"]))
+    if args.parquet:
```

Comment:
We cannot explode this. The code must re-use the template. Think how we can extend it with the **minimal** changes.

Reply:
Okay. Then I could simply add a `save_bags_hook` to the template, like is done to save the metadata. Will create a PR tomorrow in ml.